packages/typespec-ts/src/modular/buildPagingFiles.ts

```diff
@@ -124,37 +122,6 @@ export function buildPagingHelpers(
     return;
   }
 
-  const checkingPagingRequestContent = needUnexpectedHelper
-    ? `if (isUnexpected(response)) {
-      throw createRestError(
-        \`Pagination failed with unexpected statusCode \${response.status}\`,
-        response
-      );
-    }`
-    : `const Http2xxStatusCodes = [
```

Comment:
wonder if it could be an issue to change from the enumerate status codes to 200-299 range?

Reply:
Gut feeling is this would be ok, seems unlikely that any 2xx status code wouldn't be expected (or any non-2xx status wouldn't be unexpected). @joheredi you're touching these helpers in your PR too, right? IIRC you were doing the same thing (ie accepting 2xx status as ok)? Do you think it's OK?